x/gov/types/msgs_test.go

```diff
@@ -27,26 +27,28 @@ func init() {
 func TestMsgSubmitProposal(t *testing.T) {
 	tests := []struct {
 		title, description string
+		isExpedited        bool
 		proposalType       string
 		proposerAddr       sdk.AccAddress
 		initialDeposit     sdk.Coins
 		expectPass         bool
 	}{
-		{"Test Proposal", "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsPos, true},
-		{"", "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsPos, false},
-		{"Test Proposal", "", ProposalTypeText, addrs[0], coinsPos, false},
-		{"Test Proposal", "the purpose of this proposal is to test", ProposalTypeText, sdk.AccAddress{}, coinsPos, false},
-		{"Test Proposal", "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsZero, true},
-		{"Test Proposal", "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsMulti, true},
-		{strings.Repeat("#", MaxTitleLength*2), "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsMulti, false},
-		{"Test Proposal", strings.Repeat("#", MaxDescriptionLength*2), ProposalTypeText, addrs[0], coinsMulti, false},
+		{"Test Proposal", "the purpose of this proposal is to test", true, ProposalTypeText, addrs[0], coinsPos, true},
+		{"", "the purpose of this proposal is to test", false, ProposalTypeText, addrs[0], coinsPos, false},
+		{"Test Proposal", "", true, ProposalTypeText, addrs[0], coinsPos, false},
+		{"Test Proposal", "the purpose of this proposal is to test", false, ProposalTypeText, sdk.AccAddress{}, coinsPos, false},
+		{"Test Proposal", "the purpose of this proposal is to test", true, ProposalTypeText, addrs[0], coinsZero, true},
+		{"Test Proposal", "the purpose of this proposal is to test", false, ProposalTypeText, addrs[0], coinsMulti, true},
+		{strings.Repeat("#", MaxTitleLength*2), "the purpose of this proposal is to test", true, ProposalTypeText, addrs[0], coinsMulti, false},
```

Comment:
What do you think about adding both true & false for all possible test cases?

Reply:
There is no extra validation resulting from the "is expedited" flag at this time so this might be redundant. In my opinion, only if we add it in the future, does repeating all cases for true/false values would make sense.

Reply:
makes sense